content/zh/gen.go

```diff
@@ -0,0 +1,3 @@
+package zh
```

Comment:
There is a same file in the /en directory.

I copied it to /zh

Reply:
Ah sorry, my mistake. I wasn't thinking when I reviewed this. 

We don't actually need to duplicate this file. It's part of a legacy setup where we offer a redirect to the latest `master` docs on pkg.go.dev. This is not language specific. Nor is it actually required these days, because reading the docs for `master` is supported directly: https://pkg.go.dev/cuelang.org/go@master

So this file and `content/zh/pkg.go.dev.md` can be removed.

Thanks

Reply:
Deleted `gen.go` and `pkg.go.dev.md` in `zh` directory.